src/librustc/diagnostics.rs

```diff
@@ -112,6 +112,40 @@ reference when using guards or refactor the entire expression, perhaps by
 putting the condition inside the body of the arm.
 "##,
 
+E0009: r##"
+In a pattern, all values that don't implement the Copy trait have to be binded the same way
```

Comment:
Nitpick: "binded" should be "bound".

I think it's also more natural if we leave off the "than the others".